packages/systemd/9009-machine-id-setup-generate-stable-ID-under-VMWare.patch

```diff
@@ -0,0 +1,66 @@
+From f18ea74a0a62436849cf26ba9b8d5ab78d4f61a5 Mon Sep 17 00:00:00 2001
+From: Arnaldo Garcia Rincon <agarrcia@amazon.com>
+Date: Wed, 28 Apr 2021 22:23:23 +0000
+Subject: [PATCH 9009/9009] machine-id-setup: generate stable ID under VMWare
```

Comment:
This should get merged into a single patch with 9004, so we don't have two patches touching the same part of upstream code. Don't worry about preserving the attribution for me.